controllers/controllers/resource/fetcher_test.go

```diff
@@ -281,6 +282,7 @@ func TestMapMachineTemplateToCloudStackWorkerMachineConfigSpec(t *testing.T) {
 				Spec: anywherev1.CloudStackMachineConfigSpec{
 					Template:          anywherev1.CloudStackResourceIdentifier{Name: "rhel8-1.20"},
 					ComputeOffering:   anywherev1.CloudStackResourceIdentifier{Name: "large"},
+					Affinity:         "anti",
```

Comment:
you can detect lint issues with `make lint` and then execute gofumpt on the folder/file to fix them

Reply:
Fixed.